jupyterlitexblock/jupyterlitexblock.py

```diff
@@ -176,3 +189,18 @@ def studio_submit(self, request, _suffix):
         self.default_notebook = self.save_file(notebook)
         response = {"result": "success", "errors": []}
         return self.json_response(response)
+
+    @XBlock.json_handler
+    def get_completion_delay_seconds(self, data, suffix=''):
```

Comment:
istead of writing a handler to get `COMPLETION_DELAY_SECONDS ` you should pass it in context of templates inside `student_view`.

Reply:
Passed as data to the template in student view